app/mbm/templates/mbm/index.html

```diff
@@ -13,14 +13,21 @@
   <div class="row">
     <div id="controls-container" class="col-12 col-md-3">
       <div class="container-fluid m-auto form-group">
-        <div id="input-elements" class="hideable collapse multi-collapse">
-          <input id="source_text" class="form-control my-2" name="source_text" placeholder="Start address"></input>
```

Comment:
That's correct and it's a little easier to see what I'm doing here in these two commits: https://github.com/jeancochrane/mellow-bike-map/pull/41/commits/5b83114d1bf29b75058482a00dc1f53f8a2e535b and https://github.com/jeancochrane/mellow-bike-map/pull/41/commits/d28991dd61e7ee0e360158e3e40c5a79d9be7586. The first one moves the storage of the actual coordinates that get sent to the backend into our `App` object. The second one removes the `_text` bit from the id because there's no longer a second input to differentiate it from.